editor/import/scene_import_settings.cpp

```diff
@@ -657,6 +664,8 @@ void SceneImportSettings::_select(Tree *p_from, String p_type, String p_id) {
 				scene_import_settings_data->hide_options = editing_animation;
 			} else if (Object::cast_to<AnimationPlayer>(nd.node)) {
 				scene_import_settings_data->category = ResourceImporterScene::INTERNAL_IMPORT_CATEGORY_ANIMATION_NODE;
+			} else if (Object::cast_to<Skeleton3D>(nd.node)) {
+				scene_import_settings_data->category = ResourceImporterScene::INTERNAL_IMPORT_CATEGORY_SKELETON_3D_NODE;
```

Comment:
See also scene_import_settings.cpp line 343 (`SceneImportSettings::_fill_scene`) for a similar if/else statement which may need CATEGORY_SKELETON_3D_NODE added.